src/main/kotlin/net/maizegenetics/phgv2/brapi/model/Analysis.kt

```diff
@@ -0,0 +1,16 @@
+package model
+
+import kotlinx.serialization.Serializable
+import utilities.OffsetDateTimeAsStringSerializer
+import java.time.OffsetDateTime
+
+@Serializable
+data class Analysis(
```

Comment:
Is this needed for the Variants endpoint?

Reply:
This is used in VariantSet, which I pulled, but we have not yet implemented.